content/downloads.md

```diff
@@ -8,7 +8,7 @@ summary: Downloads for the Virtual Puppet Project
 
 [Stable releases - all versions](https://github.com/virtual-puppet-project/vpuppr/releases)
 
-~~Nightly release~~ Not yet provided
+[Nightly release - 0.9.0](https://github.com/virtual-puppet-project/vpuppr/actions/runs/3239018638)

```

Comment:
1. A nightly release should not have a version associated with it
2. The URL should not point to a specific build since it will be immediately out of date tomorrow

Reply:
Yeah that's probably best. I'm not aware of any url that points to the latest job for a specific workflow.